src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactoryImplTest.java

```diff
@@ -546,4 +478,111 @@ private String readNoSshRepositoryFromFile() {
     private String readProjectFromFile() {
         return readFileToString("/project-response.json");
     }
+
+    private String readWebhookCapabilitiesResponseFromFile() {
+        return readFileToString("/webhook-capabilities-response.json");
+    }
+
+    private class MockRemoteHttpServer implements Call.Factory {
```

Comment:
I like this class, but how about we move it to a proper class in the `fixture` package? It is about 100 lines of just test fixture and I think it would be good to not have it in here. The file is pretty long already.

Reply:
Since I am already refactoring client, I'll handle it in my next PR.